### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cs-shivaji-kolape**

### CODE REVIEW

File: cisco-talos-feed/info.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
+{
+  "name": "cisco-talos-feed",
+  "label": "Cisco Talos Feed",
+  "description": "Talos Reputation Center provides access to expansive threat data and related information. This connector facilitates automated operations related to pull list of blacklist IP Addresses.",
```
Comment (cs-pooja-singh) [inline cisco-talos-feed/info.json]: related to fetching the list of blacklisted IP addresses

File: cisco-talos-feed/operations.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        talos = CISCOTalosFeed(config)
+        mode = params.get('output_mode')
+        create_pb_id = params.get("create_pb_id")
+        resp = talos.make_rest_call('/documents/ip-blacklist')
+        list_indicators =  list(set(resp.split('\n') if resp else []))
+        indicators = [indicator for indicator in list_indicators if indicator]
+        if mode == 'Create as Feed Records in FortiSOAR':
+            trigger_ingest_playbook(indicators, create_pb_id, parent_env=kwargs.get('env', {}),
+                                    batch_size=1000)
+            return 'Successfully triggered playbooks to create feed records'
```
Comment (cs-pooja-singh) [inline cisco-talos-feed/operations.py:71]: the options in info.json have one more mode Save to File. you can remove that if not supported from here

File: cisco-talos-feed/operations.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+
+def fetch_indicators(config, params, **kwargs):
+    try:
+        talos = CISCOTalosFeed(config)
+        mode = params.get('output_mode')
+        create_pb_id = params.get("create_pb_id")
+        resp = talos.make_rest_call('/documents/ip-blacklist')
+        list_indicators =  list(set(resp.split('\n') if resp else []))
+        indicators = [indicator for indicator in list_indicators if indicator]
```
Comment (cs-pooja-singh) [inline cisco-talos-feed/operations.py:67]: is this to remove empty lines?

Comment (cs-shivaji-kolape) [reply cisco-talos-feed/operations.py:67]: yes, this line removes an empty element from the list.

**Pull request merged**